x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/assets.tsx

```diff
@@ -70,74 +93,70 @@ export const AssetsPage = ({ packageInfo, refetchPackageInfo }: AssetsPanelProps
 
   useEffect(() => {
     const fetchAssetSavedObjects = async () => {
-      if ('installationInfo' in packageInfo) {
-        if (spaces) {
-          const { id: spaceId } = await spaces.getActiveSpace();
-          const assetInstallSpaceId = packageInfo.installationInfo?.installed_kibana_space_id;
-
-          // if assets are installed in a different space no need to attempt to load them.
-          if (assetInstallSpaceId && assetInstallSpaceId !== spaceId) {
-            setAssetsInstalledInCurrentSpace(false);
-            setIsLoading(false);
-            return;
-          }
-        }
-
-        const pkgInstallationInfo = packageInfo.installationInfo;
+      if (!pkgInstallationInfo) {
+        setIsLoading(false);
+        return;
+      }
 
-        if (
-          pkgInstallationInfo?.installed_es &&
-          Array.isArray(pkgInstallationInfo.installed_es) &&
-          pkgInstallationInfo.installed_es.length > 0
-        ) {
-          const deferredAssets = pkgInstallationInfo.installed_es.filter(
-            (asset) => asset.deferred === true
-          );
-          setDeferredInstallations(deferredAssets);
-        }
-        const authorizedTransforms = (pkgInstallationInfo?.installed_es || []).filter(
-          (asset) => asset.type === ElasticsearchAssetType.transform && !asset.deferred
-        );
+      if (spaces) {
+        const { id: spaceId } = await spaces.getActiveSpace();
+        const assetInstallSpaceId = pkgInstallationInfo.installed_kibana_space_id;
 
-        if (
-          authorizedTransforms?.length === 0 &&
-          (!pkgInstallationInfo?.installed_kibana ||
-            pkgInstallationInfo.installed_kibana.length === 0)
-        ) {
+        // if assets are installed in a different space no need to attempt to load them.
```

Comment:
I am wondering if we could still display the ES assets in a different space, as those are not space aware e.g. if nginx was added to space A, index templates, ingest pipelines, etc. will be available in the default space too.

Reply:
I think we should try to display the ES assets regardless of the `installed_kibana_space_id` on the integration, as those assets are not tied to spaces.

Reply:
great catch! I missed this part of the original code. thanks @juliaElastic